coderd/database/dbfake/dbfake.go

```diff
@@ -685,6 +685,13 @@ func (q *FakeQuerier) GetActiveDBCryptKeys(_ context.Context) ([]database.DBCryp
 	return ks, nil
 }
 
+func minTime(t, u time.Time) time.Time {
```

Comment:
nit: if this is used only by `ActivityBumpWorkspace`, then I would move it below/to the bottom.